.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve the ohsome API.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### General Information
+ - Affected endpoint(s) [e.g. /contributions/count]
+ - Affected parameter(s) [e.g. bpolys, format]
+ - HTTP method used [e.g. POST]
+ - Used tool/library for the request [e.g. cURL, Postman]
+
+### How to Reproduce
+Steps [e.g.] to reproduce the behaviour:
+1. Start locally the ohsome API
+2. Send this request [URL] (http://localhost:8080/contributions/count?bboxes=8.67%2C49.39%2C8.71%2C49.42&time=2017-01-01%2F2017-01-02%2FP1Y)
+3. See error message 'Unable to find...'
+
+### Expected Behaviour
+A clear and concise description of what you expected to happen [e.g. "The error message should be..."]
```

Comment:
```suggestion
A clear and concise description of what you expected to happen.
```

I would drop the example here, since the expected behaviour could be so many different things. Similarly we also don't give an example for the initial _bug description_ section.